docs/development.md

```diff
@@ -75,13 +75,28 @@ To make all neccessary files for the clients, dashboard, and the work generator,
   docker-compose up -d
   ```
   
-  If you forget the `-d`, it will launch in the foreground, and you will see the log output for all containers.  Press ctlr-c to exit all containers
+  If you forget the `-d`, it will launch in the foreground, and you will see the log output for all containers. Press ctlr-c to exit all containers
+
+* Using devup to Build and Launch Docker Containers
+
+  ```
+  bash devup -b
+  ```
+
+  If you forget the `-b`, it will build the containers before launching them. Additionally you can run devup 
+with the `-b` to just launch the containers. If you run devup you don't need to build or launch the containers manually.
```

Comment:
"with" should be "without" here - but this sentence might just go away with my comments, above.